src/Features/CSharp/Portable/CodeFixes/Suppression/CSharpSuppressionCodeFixProvider.cs

```diff
@@ -110,8 +111,7 @@ protected override SyntaxNode AddGlobalSuppressMessageAttribute(SyntaxNode newRo
             bool needsLeadingEndOfLine)
         {
             var attributeArguments = CreateAttributeArguments(targetSymbol, diagnostic);
-            var attribute = SyntaxFactory.Attribute(SyntaxFactory.ParseName(SuppressMessageAttributeName), attributeArguments)
-                .WithAdditionalAnnotations(Simplifier.Annotation);
+            var attribute = SyntaxFactory.Attribute(SyntaxFactory.ParseName(SuppressMessageAttributeName), attributeArguments);
```

Comment:
get symbol for suppress attribute (using GetSymbolBymetadata ...) and symbol has ToMinimallyQualifed ... method in it.

also, for global suppression, since you are putting all those attribute to same context, get minimallyQualifed string once, and reuse them.